### PR TITLE
docs: add tested OS

### LINKED ISSUE #53: Add supported OS
Only Centos 7 and Rocky 8 are supported, we should add it inside the README.

### PR DESCRIPTION
**Pull request by rpignolet**
Fix #53

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,15 @@ Versions are approximately based on the [HDP 3.1.5 release](https://docs.clouder
 
 **Note:** A project can graduate from "TDP Extras" to "TDP Core" if enough people are supporting it and/or if it is made compatible with all the other projects of the stack.
 
+## Supported operating system (OS)
+
+Only bare metal and virtual machine deployment are supported. Container based OS may work but not guarantee.
```
Comment (gboutry) [inline README.md:49]:
```suggestion
Only bare metal and virtual machine deployment are supported. Container based OS may work but are not guaranteed.
```

File: README.md [hunk 2]
```diff
@@ -44,6 +44,15 @@ Versions are approximately based on the [HDP 3.1.5 release](https://docs.clouder
...
+
+- Centos 7
+- Rocky 8
+
+Redhat like OS may work but not guarantee.
```
Comment (gboutry) [inline README.md:54]:
```suggestion
Redhat like OS may work but are not guaranteed.
```

**Review by mehdibn — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -45,6 +45,15 @@ Versions are approximately based on the [HDP 3.1.5 release](https://docs.clouder
 
 **Note:** A project can graduate from "TDP Extras" to "TDP Core" if enough people are supporting it and/or if it is made compatible with all the other projects of the stack.
 
+## Supported operating system (OS)
```
Comment (rpignolet) [inline README.md:48]: Change word "supported" to "tested".

Comment (rpignolet) [reply README.md]: done

**Review by gboutry — APPROVED** [review]

**Review by leopaul36 — APPROVED** [review]

**Pull request merged**